README.md

```diff
@@ -45,11 +45,11 @@ After you're finished please remove all the comments and instructions!
   - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
-  <!--- [Prerequisites](#prerequisites)
+  <!--- [Prerequisites](#prerequisites)-->
```

Comment:
- [ ] As the previous reviewer suggested kindly work on your delete button. If I click middle or left side of delete button it's not responding, till I click the right top corner of the button. Please look throught the video and fix the issue thank you!


https://user-images.githubusercontent.com/69011963/220871635-81a16b74-cf3e-4cd7-93fa-70b67002714d.mov